s3/pkg/datastore/azure/azure.go

```diff
@@ -46,6 +46,48 @@ type AzureAdapter struct {
 	containerURL azblob.ContainerURL
 }
 
+func (ad *AzureAdapter) BucketDelete(ctx context.Context, input *pb.Bucket) error {
+
+	containerURL, _ := ad.createBucketContainerURL(ad.backend.Access, ad.backend.Security, input.Name)
```

Comment:
In createBucketContainerURL, make first char capital. i.e CreateBucketContainterURL()